searx/templates/simple/result_templates/images.html

```diff
@@ -1,6 +1,6 @@
 <article class="result result-images {% if result['category'] %}category-{{ result['category'] }}{% endif %}">{{- "" -}}
         <a {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} href="{{ result.img_src }}">{{- "" -}}
-                <img class="image_thumbnail" {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} src="{% if result.thumbnail_src %}{{ image_proxify(result.thumbnail_src) }}{% else %}{{ image_proxify(result.img_src) }}{% endif %}" loading="lazy" alt="{{ result.title|striptags }}">{{- "" -}}
+                <img class="image_thumbnail" {% if results_on_new_tab %}target="_blank" rel="noopener noreferrer"{% else %}rel="noreferrer"{% endif %} src="{% if result.thumbnail_src %}{{ image_proxify(result.thumbnail_src) }}{% else %}{{ image_proxify(result.img_src) }}{% endif %}" alt="{{ result.title|striptags }}" loading="lazy" width="200" height="200">{{- "" -}}
```

Comment:
AFIU, the `height="200"` matches the 200 here:
https://github.com/searxng/searxng/blob/5307fd841d11d0bead4ef07147b46e584b3b1fe4/searx/static/themes/simple/src/js/main/results.js#L6

Reply:
That is correct. Since width is not know until image is loaded, I chose 200x200.

Reply:
See my comments below: we can avoid random value since the CSS will set explicitly the width in rem value:
https://github.com/searxng/searxng/blob/9fc27088ef9cbbbc09b07a1b60cc78b469b1d0f3/searx/static/themes/simple/src/less/style.less#L265-L280

It is not always true for `.thumbnail`, for tablet & phone the width is changed:
https://github.com/searxng/searxng/blob/9fc27088ef9cbbbc09b07a1b60cc78b469b1d0f3/searx/static/themes/simple/src/less/style.less#L677-L679

but it will will the right width at in some cases (and, _not related to this PR_, don't look the web pages without CSS).

Otherwise, LGTM.